internal/pkg/cli/job_init_test.go

```diff
@@ -65,7 +66,7 @@ func TestJobInitOpts_Validate(t *testing.T) {
 			setupMocks: func(m initJobMocks) {
 				m.mockStore.EXPECT().GetApplication("phonetool").Return(&config.Application{}, nil)
 			},
-			wantedErr: errors.New("open hello/Dockerfile: file does not exist"),
+			wantedErr: errors.New("open " + filepath.FromSlash("hello/Dockerfile") + ": file does not exist"),
```

Comment:
nit (here and elsewhere):
```suggestion
			wantedErr: errors.New("open %s: file does not exist", filepath.FromSlash("hello/Dockerfile")),
```

Reply:
Should be fixed!